vms/proposervm/vm.go

```diff
@@ -353,17 +386,38 @@ func (vm *VM) SetPreference(ctx context.Context, preferred ids.ID) error {
 	// custom VMs. Until the P-chain is modified to target a specific block
 	// time, ProposerMinBlockDelay can be configured in the subnet config.
 	minDelay = math.Max(minDelay, vm.MinBlkDelay)
+	nextStartTime := parentTimestamp.Add(minDelay)
 
-	preferredTime := blk.Timestamp()
-	nextStartTime := preferredTime.Add(minDelay)
-	vm.Scheduler.SetBuildBlockTime(nextStartTime)
+	return nextStartTime, nil
+}
 
-	vm.ctx.Log.Debug("set preference",
-		zap.Stringer("blkID", blk.ID()),
-		zap.Time("blockTimestamp", preferredTime),
-		zap.Time("nextStartTime", nextStartTime),
+func (vm *VM) resetPostDurangoScheduler(
```

Comment:
Can we rename this (and the above `resetPreDurangoScheduler`) to something more accurate? `getPostDurangoSlotTime`? or something? This doesn't actually `reset` anything.

Reply:
Done